examples/contracts/custom/src/cw1.rs

```diff
@@ -1,11 +1,14 @@
-use cosmwasm_std::{CosmosMsg, Response, StdError, StdResult};
+use cosmwasm_std::{CosmosMsg, Empty, Response, StdError, StdResult};
 use cw1::{CanExecuteResp, Cw1};
 use sylvia::types::{ExecCtx, QueryCtx};
 
 use crate::contract::CustomContract;
 
 impl Cw1 for CustomContract {
     type Error = StdError;
+    type ExecC = Empty;
+    type QueryC = Empty;
+    type CosmosCustomMsg = Empty;
```

Comment:
What is this one about? @jawoznia do we need this for anything? I see it in more places so maybe I miss something?

Reply:
`ExecC` and `QueryC` are special associated types by which you can define the `CustomMsg` and `CustomQuery` types.

The `CosmosCustomMsg` is not needed here judging by the name.

Reply:
As discussed in Slack, it seems needed for now since it wouldn't compile.